zone/client.h

```diff
@@ -2429,6 +2450,70 @@ class Client : public Mob
 	bool IsFilteredAFKPacket(const EQApplicationPacket *p);
 	void CheckAutoIdleAFK(PlayerPositionUpdateClient_Struct *p);
 	void SyncWorldPositionsToClient(bool ignore_idle = false);
+
+
+ 	Mob* GetMob() {
+ 		return Mob::GetMob();
+ 	}
+
+ 	void Clone(Client& in)
```

Comment:
What is this for ?

Reply:
I see it in another part of the PR, for the offline functionality

Reply:
It is a helper to clone a client